contrib/docker/docker-compose.yml

```diff
@@ -1,4 +1,8 @@
 # docker-compose build && docker-compose up
+# If you experience problems with the CKAN container not being
+# able to connect to the DB, then most likely the DB has not
+# started up quickly enough. Just do "docker-compose up ckan"
+# again to retry
```

Comment:
@Cdower We don't have that in docker-compose file format version 1. There's actually no good reason not to move to version 2 or 3, but I was originally trying to make a minimal change and didn't want to take that step.

Beyond that though, I'm not sure depends_on will help anything. Docker explicitly state: "However, Compose will not wait until a container is “ready” (whatever that means for your particular application) - only until it’s running." - https://docs.docker.com/compose/startup-order/